src/main/java/com/booleanuk/extension/Extension.java

```diff
@@ -6,6 +6,10 @@ public class Extension {
     public void stepOne() {
         // TODO: 1. Write a for loop that adds all the even numbers between 0 and 6 (0, 2, 4, 6) to evenNums
 
+        numsEven = new int[4];
+        for(int i = 0; i <= 6; i += 2) {
+            numsEven[i/2] = i;
```

Comment:
There is also a way to do this making use of the modulo operator `%`

Reply:
True, normaly i would have used mod and looped through all numbers until 6 but Dave presented this way for even numbers, which lets you skip odd numbers all together. 

What would be the preferred method if there is one?

Reply:
It's probably a personal preference in this case. As a rule, it's best practice if you're unsure to aim for readability. Matching any conventions of the codebase you are working on.